.env.development

```diff
@@ -1,3 +1,3 @@
-FIND_BASE_URL=https://api2.publish-teacher-training-courses.service.gov.uk/api/v3/
+FIND_BASE_URL=https://api.publish-teacher-training-courses.service.gov.uk/api/v3/
```

Comment:
Don't the `.env` files stack up and override each other, and we could lose this here? https://github.com/bkeepers/dotenv#can-i-use-dotenv-in-production

Reply:
Just tried removing this locally and it worked ok